src/concrete/ml/sklearn/linear_model.py

```diff
@@ -663,132 +709,6 @@ def fit(  # type: ignore[override]
             sample_weight=sample_weight,
         )
 
-    # pylint: disable-next=too-many-branches,too-many-statements
-    def _fit_encrypted_one_step(self, X, y, fhe):
```

Comment:
as I mentioned above, this is no longer needed (and it removes a lot of duplicated code)